packages/mobile/package.json

```diff
@@ -8,7 +8,7 @@
     "lint": "tslint -c tslint.json --project tsconfig.json",
     "build": "yarn run build:ts && yarn run build:metro",
     "build:ts": "tsc --noEmit",
-    "build:metro": "echo 'NOT WORKING RIGHT NOW'",
+    "build:metro": "yarn run react-native bundle --entry-file index.js --bundle-output bundle.js; rm bundle.js",
```

Comment:
Nice! I'm wondering if we should remove the mobile-test-build-app job in CI if we have this working now. They're not exactly the same but there's a decent amount of overlap there. 
https://github.com/celo-org/celo-monorepo/blob/master/.circleci/config.yml#L846

Reply:
Yeah good call